config/v1/types_infrastructure.go

```diff
@@ -1082,9 +1082,42 @@ type VSpherePlatformStatus struct {
 	LoadBalancer *VSpherePlatformLoadBalancer `json:"loadBalancer,omitempty"`
 }
 
+// IBMCloudServiceEndpoint stores the configuration of a custom url to
+// override existing defaults of IBM Cloud Services.
+type IBMCloudServiceEndpoint struct {
+	// name is the name of the IBM Cloud service.
+	// For example, the IBM Cloud Private IAM service could be configured with the
+	// service `name` of `Iam` and `url` of `https://private.iam.cloud.ibm.com`
```

Comment:
Why are `Iam` and `Vpc` like that rather than `IAM` and `VPC`? Is this a requirement or IBM somehow? Is it possible we could use the full caps version of these acronyms as this would be consistent with Kube conventions

Reply:
I actually started with `IAM` and `VPC` (then had lowercase), but converted them, per your request above to PascalCase
https://github.com/openshift/api/pull/1499#discussion_r1247561353

I can use either, so it doesn't matter to me. I'll switch them back to all caps for now.

Reply:
PascalCase for these would be IAM and VPC since they are acronyms